src/components/tl/ucp/reduce_scatter/reduce_scatter_ring.c

```diff
@@ -12,6 +12,8 @@
 #include "utils/ucc_coll_utils.h"
 #include "utils/ucc_dt_reduce.h"
 
+#define REVERSED_FRAG 1
```

Comment:
Can you explain what is the point of such a variable ? Do we want to make this flag configurable ?

Reply:
no its not configurable I added this more for readability. For example when we check if (frag == 1) this ight be confusing and it is better understood what this if statement is checking when using if (frag == REVERSED_FRAG).

This is my opinion, but if you still think this should be removed let me know,.

Reply:
Thanks for the explanation
In my opinion, it doesn't make it clearer because it introduces a new variable, and `if (frag) {...}` is just enough
But it's not critical, so please do what you think is best